kitchen.yml

```diff
@@ -14,24 +14,29 @@ verifier:
   load_plugins: true
 
 provisioner:
-  name: puppet_apply
-  sudo: true
+  name: ansible_playbook
+  hosts: all
   require_chef_for_busser: false
-  manifests_path: ./spec/puppet/manifests
-  modules_path: ./spec/puppet/modules
-  hiera_config_path: ./spec/puppet/hiera.yaml
-  hiera_data_path: ./spec/puppet/hiera
-  resolve_with_librarian_puppet: true
-  ignore_spec_fixtures: true
+  require_ruby_for_busser: false
+  require_pip3: true
+  ansible_binary_path: /usr/local/bin
+  ansible_verbose: true
+  roles_path: spec/ansible/roles
+  galaxy_ignore_certs: true
+  ansible_extra_flags: <%= ENV['ANSIBLE_EXTRA_FLAGS'] %>
```

Comment:
Is this env we have to set if we want to use it documented in the readme?

Reply:
Updated readme to document usage of this environmental variable

Reply:
Suggest we strip out the references to Beaker and Puppet, since we are no longer using them for the hardening content @HenryXiaoHX 

Also, please take a look at the "Testing with Kitchen" section of the README -- might need an update now.

We will likely need to look at updating the README in general as we update the profile, but for now we can focus on the kitchen testing content.